app/frontend/src/views/About.vue

```diff
@@ -1,69 +1,70 @@
 <template>
```

Comment:
i added v-containers to this page when i was experimenting with having a v-container in the header/navbar.. so that it would line up. Now that Mina is suggesting padding for the header/navbar instead, i can revert changes to the about page if you like. but it does look a little better in my opinion, especially as we have that video player which gets a little wide.